l1-contracts/contracts/bridge/L1ERC20Bridge.sol

```diff
@@ -140,7 +140,7 @@ contract L1ERC20Bridge is IL1ERC20Bridge, ReentrancyGuard {
             chainId: ERA_CHAIN_ID,
             l2BatchNumber: _l2BatchNumber,
             l2MessageIndex: _l2MessageIndex,
-            l2Sender: l2Bridge,
+            l2Sender: L1_NULLIFIER.__DEPRECATED_l2BridgeAddress(ERA_CHAIN_ID),
```

Comment:
it works, but maybe to not spread the usage of deprecated methods, it makes sense to create a separate method for it?